### ISSUE TITLE
Example suited Kubernetes environment unclear

### ISSUE THREAD

**Issue by maxwell92** [openshift-evangelists/kbe#26]
On the front page we should tell those examples below are expected to run on Kubernetes version 1.x.x recommended.  To reduce the confusion when something goes wrong.

**Comment by maxwell92** [comment]
Like I just run the command below on my Kubernetes 1.4.9 
```shell
$ kubectl run sise --image=mhausenblas/simpleservice:0.5.0 --port=9876
```
But I never get a running Pod and the RS told me that:

```shell
Error creating: pods "sise-3210265840-" is forbidden: failed quota: default-quota: must specify cpu,memory
```

Is it a version-related problem ? or I just misconfigured ?

**Comment by mhausenblas** [comment]
I agree that it would improve the UX if this info is prominently displayed on the landing page. The setup (with the Kube version used) is described currently here: http://kubernetesbyexample.com/diy/ and only linked from the landing page. I'll address this issue when updating the entire site to 1.7.
